libraries/jevmodal/media/css/jevmodal.css

```diff
@@ -161,7 +162,7 @@
 	max-height:100%;
 }
 .jevmodal .modal-dialog {
-    height: 100%;
+    height: 95%;
```

Comment:
It's indeed! The site is Kenneth's testing site, let me know if you need site access.